File: 26-RemoveDuplicatesFromSortedArray/26-RemoveDuplicatesFromSortedArray.java
// Last updated: 1/3/2026, 6:27:08 PM
class Solution {
    public int removeDuplicates(int[] nums) {
        int k = 0, n = nums.length;
        for(int i = 0; i < n; i++){
            if(i == 0 || nums[i] != nums[i-1]){
                nums[k] = nums[i];
                k++;
            }
        }
        return k;
    }
}

File: 26-RemoveDuplicatesFromSortedArray/26-RemoveDuplicatesFromSortedArray.cpp
// Last updated: 1/3/2026, 6:27:05 PM
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int i = 0; 
        for(int j = 1; j < nums.size(); j++){
            if(nums[j] != nums[i]){
                nums[i+1] = nums[j];
                i++;
            }
        }
        return (i+1);
    }
};